package.json

```diff
@@ -14,10 +14,12 @@
   },
   "scripts": {
     "create-admin": "node_modules/@babel/node/bin/babel-node.js src/app/scripts/create.first.admin",
+    "migrate": "node_modules/db-migrate/bin/db-migrate up",
     "test": "nyc mocha --require @babel/register './__tests__/**/**spec.js'",
     "test:watch": "mocha --watch",
     "lint": "eslint \"src/**/*.js\"",
-    "start": "nodemon --exec node_modules/@babel/node/bin/babel-node.js index.js"
+    "start:dev": "nodemon --exec node_modules/@babel/node/bin/babel-node.js index.js",
+    "start": "babel ./ --out-dir dist --ignore node_modules,babel.config.js,.github,jest.config.js && node dist/index.js"
```

Comment:
Hi @sikiru-moshood , I think this will still fail on Heroku because your using babel, babel is a dev dependency, What we can do here is add a  script that should build the project, Heroku would run this script before deployment, then your start script should just point node to the build script. here take a look
```
"clean": "rm -rf build && mkdir build",
 "build-babel": "babel -d ./build ./src -s",
  "build": "npm run clean && npm run build-babel",
   "start": "node ./build/index.js",
```

Reply:
You are right. Just added a build script